ISICbalance/ViewModels/BalanceViewModel.swift

```diff
@@ -25,13 +25,13 @@ class BalanceViewModel: BaseViewModel {
             return SignalProducer<Balance, RequestError>(error: RequestError.actionError)
         }
     }
-    
+
     init(_ requestManager: RequestManager) {
         self.requestManager = requestManager
         super.init()
         
-        requestManager.currentBalance.producer.skipNil().startWithValues { [unowned self] user in
-            self.balance.value = user.balance
+        self.getBalanceAction.values.producer.startWithValues { balance in
+            self.balance.value = balance.balance
```

Comment:
Nooo a krom toho, že tu máš memory leak (chybí weak self), tak nešlo by tohle napsat jako `self.balance <~ getBalanceAction.values`?

Reply:
A nemít to jako MutableProperty, ale rovnou jen property s initial: 0, then: values? 😏

Reply:
K tomu weakselfu nemám co dodat... 🤦‍♂️ 😞
Co se týče toho řešení s property, tak to jsem neměl tušení, že něco takového lze. 
Ještě si musím ten reactive swift pořádně osahat...